java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Appetizer/AppetizerRepository.java

```diff
@@ -1,16 +1,69 @@
 package com.my.FoodTruckApp2.Appetizer;
 
+import com.my.FoodTruckApp2.Customer.Customer;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.http.HttpStatus;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
 import org.springframework.stereotype.Repository;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.server.ResponseStatusException;
 
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.List;
+
 @Repository
+@RequiredArgsConstructor
+@Slf4j
 public class AppetizerRepository {
-    private final FoodTruck2 appetizer1 = new FoodTruck2(1,"Garlic bread","18/22","21/22",3);
-    private final FoodTruck2 appetizer2 = new FoodTruck2(2,"Breadsticks ","11/22","28/22",2);
-    ArrayList<FoodTruck2> appetizers = new ArrayList<>(Arrays.asList(appetizer1,appetizer2));
+    @Autowired
```

Comment:
If you use private final here, you should not need to auto wire this bean

Reply:
Done.